src/components/Icon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Icon = ({ onclick, id, svg }) => {
+    return <div className="toolbar-icon" onClick={ () => onclick(id)} >
+        { svg }
+    </div>
+}
+
+export default Icon
```

Comment:
This is too small to be its own component now. Let's inline it in `Toolbar`.

Reply:
Got it